app/assets/javascripts/components/course_creator/course_creator.jsx

```diff
@@ -448,101 +272,41 @@ const CourseCreator = createReactClass({
-                {language}
-                {project}
-                {privacyCheckbox}
-              </div>
-              <div className="column">
-                <DatePicker
-                  id="course_start"
-                  onChange={this.updateCourseDates}
-                  value={this.props.course.start}
-                  value_key="start"
-                  required
-                  editable
-                  label={CourseUtils.i18n('creator.start_date', this.state.course_string_prefix)}
-                  placeholder={I18n.t('courses.creator.start_date_placeholder')}
-                  blank
-                  isClearable={false}
-                  showTime={this.state.use_start_and_end_times}
-                />
-                <DatePicker
-                  id="course_end"
-                  onChange={this.updateCourseDates}
-                  value={this.props.course.end}
-                  value_key="end"
-                  required
-                  editable
-                  label={CourseUtils.i18n('creator.end_date', this.state.course_string_prefix)}
-                  placeholder={I18n.t('courses.creator.end_date_placeholder')}
-                  blank
-                  date_props={dateProps.end}
-                  enabled={!!this.props.course.start}
-                  isClearable={false}
-                  showTime={this.state.use_start_and_end_times}
-                />
-                {eventCheckbox}
-                <span className={eventClass}>
-                  {timelineStart}
-                  {timelineEnd}
-                </span>
-                {this.state.use_start_and_end_times ? timeZoneMessage : null}
-                <span className="text-input-component__label"><strong>{CourseUtils.i18n('creator.course_description', this.state.course_string_prefix)}:</strong></span>
-                <TextAreaInput
-                  id="course_description"
-                  onChange={this.updateCourse}
-                  value={this.props.course.description}
-                  value_key="description"
-                  required={descriptionRequired}
-                  editable
-                  placeholder={CourseUtils.i18n('creator.course_description_placeholder', this.state.course_string_prefix)}
-                />
-                {roleDescription}
-              </div>
-            </div>
+            <ReuseExistingCourse
+              selectClassName={selectClassName}
+              courseSelect={reuseCourseSelect}
+              options={options}
+              useThisClassAction={this.useThisClass}
+              cloneThisLabel={CourseUtils.i18n('creator.clone_this', this.state.course_string_prefix)}
+              cancelCloneAction={this.cancelClone}
+              cancelLabel={CourseUtils.i18n('cancel', this.state.course_string_prefix)}
+            />
+            <CourseForm
+              courseFormClass={courseFormClass}
+              course_utils={CourseUtils}
```

Comment:
`CourseUtils` is just an import, so there's no reason to pass it down as a prop rather than import it in the child component.